docker/openproblems-python-extras/README.md

```diff
@@ -12,6 +12,7 @@ Python packages:
 * pybedtools
 * pyensembl
 * magic-impute
+* meld
```

Comment:
this looks like the wrong PR for this, no? Not a differential abundance task PR to add this method?